config/routes.rb

```diff
@@ -1,10 +1,13 @@
+
 Rails.application.routes.draw do
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-   root 'welcome#index'
-
+  root 'cards#index'
+  get '/all' => 'cards#all'
+  #get  '/check' => 'cards#check'
+  post '/' => 'cards#check'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.